hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -58,8 +58,8 @@
 
   private final HiveSyncConfig cfg;
   private HoodieHiveClient hoodieHiveClient = null;
-  private final String snapshotTableName;
-  private final Option<String> roTableTableName;
+  private String snapshotTableName = null;
+  private Option<String> roTableTableName = null;
```

Comment:
what about `roTableName`?

Reply:
I think it’s better to keep the original name

Reply:
guess it's a typo. Probably we can fix it while you are at it.